src/ImageViewer/resources/translations/imageviewer_zh_TW.ts

```diff
@@ -0,0 +1,1631 @@
+    <message>
+        <location filename="../../src/GUI/Dialogs/SettingsDialog.cpp" line="84"/>
+        <source>Top</source>
+        <comment>ToolBarPosition</comment>
+        <translation>上方</translation>
+    </message>
+    <message>
+        <location filename="../../src/GUI/Dialogs/SettingsDialog.cpp" line="85"/>
+        <source>Left</source>
+        <comment>ToolBarPosition</comment>
+        <translation>左邊</translation>
+    </message>
+    <message>
+        <location filename="../../src/GUI/Dialogs/SettingsDialog.cpp" line="86"/>
+        <source>Right</source>
+        <comment>ToolBarPosition</comment>
+        <translation>右邊</translation>
+    </message>
+    <message>
+        <location filename="../../src/GUI/Dialogs/SettingsDialog.cpp" line="87"/>
+        <source>Movable</source>
+        <comment>ToolBarPosition</comment>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../src/GUI/Dialogs/SettingsDialog.cpp" line="162"/>
+        <source>Select Background Color</source>
+        <translation>背景色</translation>
+    </message>
+    <message>
+        <location filename="../../src/GUI/Dialogs/SettingsDialog_p.h" line="111"/>
+        <source>General</source>
+        <translation>常規</translation>
+    </message>
+    <message>
+        <location filename="../../src/GUI/Dialogs/SettingsDialog_p.h" line="112"/>
+        <source>Interface</source>
+        <translation>界面</translation>
+    </message>
+    <message>
+        <location filename="../../src/GUI/Dialogs/SettingsDialog_p.h" line="113"/>
+        <source>Decoders</source>
+        <translation>編碼</translation>
+    </message>
+    <message>
+        <location filename="../../src/GUI/Dialogs/SettingsDialog_p.h" line="116"/>
+        <source>Automatically check for updates</source>
+        <translation>自動升級</translation>
+    </message>
+    <message>
+        <location filename="../../src/GUI/Dialogs/SettingsDialog_p.h" line="120"/>
+        <source>Ask before deleting images</source>
+        <translation>刪除檔案前需要確認</translation>
+    </message>
+    <message>
+        <location filename="../../src/GUI/Dialogs/SettingsDialog_p.h" line="123"/>
+        <source>Move deleted images to trash</source>
+        <translation>刪除檔案到回收桶</translation>
+    </message>
+    <message>
+        <location filename="../../src/GUI/Dialogs/SettingsDialog_p.h" line="126"/>
+        <source>Use smooth image rendering</source>
+        <translation>平滑圖像渲染</translation>
+    </message>
+    <message>
+        <location filename="../../src/GUI/Dialogs/SettingsDialog_p.h" line="129"/>
+        <source>Allow fit to window for small images</source>
+        <translation>小圖像貼合窗口</translation>
+    </message>
+    <message>
+        <location filename="../../src/GUI/Dialogs/SettingsDialog_p.h" line="132"/>
+        <source>Remember flip and rotate during session</source>
+        <translation type="unfinished"></translation>
```

Comment:
Do you have any ideas about "Remember flip and rotate during session" translation?

Reply:
* "during session": Do that means it won't remenber flip and rotate after exiting app?
*  I can't find this option in my latest release app

Reply:
This behavior was non-switchable in previous releases. You can try rotate file then switch next and then switch back to rotated file in your previous version. It will still appear rotated until app exit. This is useful in some cases, but it is not expected and can lead to confusion for some users. Starting with version 1.7, this will be configurable by this option in settings dialog

Reply:
done

        <source>Remember flip and rotate during session</source>
        <translation>程式開啟期間，記住翻轉跟旋轉</translation>